main/ui/src/main/resources/localization_fr.properties

```diff
@@ -0,0 +1,98 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2016 Markus Kreusch
```

Comment:
Could you please tell me how to say those in german :

Weak password
Average password
Strong password


2016-04-10 16:54 GMT+02:00 Jean-Noel Charon <jncharon@gmail.com>:

> FYI I've also adjusted several strings to fit the interface
>
> 2016-04-10 16:26 GMT+02:00 Sebastian Stenzel <notifications@github.com>:
>
>> In main/ui/src/main/resources/localization_fr.properties
>> <https://github.com/cryptomator/cryptomator/pull/227#discussion_r59134848>
>> :
>>
>> > @@ -0,0 +1,98 @@
>> > +#-------------------------------------------------------------------------------
>> > +# Copyright (c) 2016 Markus Kreusch
>>
>> Put your name here, as this is your work. I know this is kind of
>> redundant to the "Contributors" list, but this line indicates who did the
>> most part.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/cryptomator/cryptomator/pull/227/files/c9d970955ca015b3c30478134caa7fea6503491d#r59134848>
>>
>
>